### DISCUSSION TITLE
Coloring of BigNumber according to thresholds

### DISCUSSION THREAD

**Discussion by gbusch** [apache/superset#22667] [Ideas]
We needed to visually indicate that the metric displayed as BigNumber is above a certain threshold, for example by coloring the number.

I have a draft implementation (see screenshot below).

Is this feature interesting enough for the community that it makes sense to open a PR or was this idea rejected for some reason in the past?

<img width="1144" alt="Screenshot 2022-10-15 at 14 35 27" src="https://user-images.githubusercontent.com/16560193/195986707-d8c7e262-9733-42b5-8cd7-74f640b3c407.png">

**Comment by cwegener** [comment]
Sounds like a good feature to me.
cc @rusackas for visibility

**Comment by rusackas** [comment]
Totally agreed, I think people would welcome this! Small caveat that I think we should make conditional formatting popovers as consistent as possible. If you look at the one from the Pivot Table, it's similar, but only has one `target value` field, since the `left` value would be the (dynamic) metric value.

<img width="450" alt="image" src="https://user-images.githubusercontent.com/812905/211654928-199d0747-f3a5-4609-981e-34f8f18147fa.png">

@gbusch I'd say go for it! Let us know here or on Slack if we can help steer things along! Meanwhile, I'll close this as an "Issue" and move it over to a Github Discussion in the Ideas section in case others want to weigh in more.

**Comment by chrmei** [comment]
Just one idea: if different colours for the number are allowed here, something like an optional text-stroke ( border ) for the number would be quite good, e.g. brighter yellow on white might not be visible really.
So, color of the border is either dependend on the choosen / background color or is a fixed color. Here a simple example ( which is not perfect ), but I think you get the idea :) 
![image](https://user-images.githubusercontent.com/12625004/214005517-09b3e746-8658-4ea4-9f84-30880682c02c.png)

**Comment by janaperedni** [comment]
why hasn't it been merged?
